stages/org.osbuild.oscap.remediation

```diff
@@ -61,63 +63,76 @@ SCHEMA = """
         "type": "string",
         "description": "The tailoring id"
       },
-      "xml_results": {
+      "arf_results": {
```

Comment:
AFAICT, this means that the `xml_results` stage options key is no longer accepted by the schema, which is backward incompatible. If it is a synonym for `xccdf_results`, you need to use `anyOf` in the schema.

Reply:
Yep, that's the idea. Hope I got it right now.